packages/commonwealth/client/scripts/state/index.ts

```diff
@@ -58,9 +55,6 @@ export interface IApp {
   threads: ThreadsController;
   threadUniqueAddressesCount: ThreadUniqueAddressesCount;
   comments: CommentsController;
-  reactions: ReactionsController;
```

Comment:
Some of this `reactions` controller state is merged with comments. We can remove this when the comments controller is migrated to react query.

The reason why it was not completely migrated now is because it uses the old store pattern which should be properly migrated to react query, and the ideal/proper way is to properly move comments state to react query and then merge this left over reactions state to the updated comments react query state